lib/renewablepotentialslib/shape_utils.py

```diff
@@ -0,0 +1,257 @@
+"""Module containing utilities."""
+import math
+
+import numpy as np
+import geopandas as gpd
+import numpy as np
+import shapely
+#import rasterio
+import pycountry
+
+
+from renewablepotentialslib.conversion import transform_bounds, eu_country_code_to_iso3
+from renewablepotentialslib import EPSG_3035, EPSG_3035_PROJ4, WGS84, WGS84_PROJ4
+
+
+def determine_pixel_areas(crs, bounds, resolution):
+    """Returns a raster in which the value corresponds to the area [km2] of the pixel.
+
+    This assumes the data comprises square pixel in WGS84.
+
+    Parameters:
+        crs: the coordinate reference system of the data (must be WGS84)
+        bounds: an object with attributes left/right/top/bottom given in degrees
+        resolution: the scalar resolution (remember: square pixels) given in degrees
+    """
+    # the following is based on https://gis.stackexchange.com/a/288034/77760
+    # and assumes the data to be in WGS84
+    #assert crs == rasterio.crs.CRS.from_epsg("4326") # WGS84
```

Comment:
This check seems necessary.

Reply:
In general for this entire PR: I hope you did not alter the functions, because this PR breaks Git history and so I have no clue what you changed. If you do need to change things, it would be good to do that in a consecutive PR.

Reply:
The only change here was to hide that check, which I only did when trying to stop my pip install from being unhappy (which ultimately didn't work). Since these utils only need rasterio as a dependency for this one assertion, I would be in favour of removing it to avoid any need for rasterio in the dependencies (for now).

Reply:
Sounds reasonable. That `crs` check is still necessary. In fact, I think we check `crs` not often enough. A lot works through implicit assumptions.

Reply:
Yeah, agreed that more places in the process should align the crs (if necessary) before continuing. Many of the dependencies do warn on this though (e.g. geopandas)